src/chigame/users/views.py

```diff
@@ -41,3 +45,13 @@ def get_redirect_url(self):
 
 
 user_redirect_view = UserRedirectView.as_view()
+
+# Gets and serializes the users
+class UserListView(generics.ListAPIView):
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
+
+# Display view for users
+def user_list(request):
```

Comment:
Why is this view here? It is not used anywhere besides the one URL path that has to be commented out for your actual Django Rest Framework endpoint to work. 

To my understanding, it is not relevant to the associated task which is creating a DRF endpoint where you can GET the list of users in Chigame.

If this is not relevant to your actual API endpoint, please remove.

Reply:
This view has been updated to show individual users and a list of all users in the database